src/smallcommands.ts

```diff
@@ -89,6 +89,29 @@ function toggleFileLint(arg) {
     }
 }
 
+async function openJuliaHelp() {
+    let searchString = await vscode.window.showInputBox()
+    if (g_languageClient == null) {
+        vscode.window.showErrorMessage('Error: Lanuage server is not yet running.');
+    }
+    else {
+        try {
+            g_languageClient.sendRequest("julia/help", searchString).then((v :string)=>{
+                vscode.window.showInformationMessage(v)
```

Comment:
We could also open this in a dedicated help pane?

Reply:
Yep, as is this just spits out the documentation (markdown markings and all) into a tiny really un-useful panel - I assume a WebView pane would be able to handle markdown?

Reply:
Hm, not sure, I think WebView only takes HTML... But there must be some facility to compile markdown into HTML in the VS Code code somewhere :)